packages/wallets/keystone/src/adapter.ts

```diff
@@ -0,0 +1,143 @@
ISn332Ga5fvy6YznytAuSwAzwPgPO4Rjkvtq5DDsSR8xrmMs/93/lLIaWNGjwJcXfMQ9dvch+83lCsBw8dOtRrmxj+5/7C3nvvPVRXVzPDHn74YdTV1eHtt9/G4sWLMWvWLBQUFODNN99EbW0tCgsLBcuz3fw/dFx6AyDX6dPnF+T6TwCIwBHB7Vrp2xd62tj0eYI2ygXFBI6KiuoVjiWgmOCA+DScxx9/HHv37sXgwYOZ4UlJSdizZw+ee+45wfzbvn8RnTXvOTdcvcypg7P1nT3UGUZwbhNHzh7LedyZhpA2KlH5qUSKCSz0BnVfYYUOgk2bNjFPsyNHjsS7774LnucF03IcB57nUVJSInKKd6D19N9w48pOEPU1vudp1+0O8qmBh1DUFyeYtA7lnSCKXYN///13AN6NTwLvcQa8r8MXLlzAhx9+yMx327ZtiIvz/ryx2FBjcXExOI4TePU/ofX03wAQEsf+h6sz+lxVPRTrfRwWPPkElrannX35kROKCXzp0iXmfl/BWeFFRUWw2fyP7pkzZ6KgoEA0rS84jkNxcTEAiIj8NECERON/OtOgz3lBHAeQt3Ce3gOOI8/ggNKa6g8w7JEXijk6dDodmpqakJKSElC6AwcO9Irom9/Zs2cxeXJwz48OhwPPP/883njjDYEYHFLuftspMue+G/Z2Zvmn6IsjtF8obfdvh9D0938LoAbBQQdgsxIZExGmT5+OcePGSU7T3d2NBQsWoL293S9s1apVWLVqVdD2cByH/Px8mEwmfPvtt+zyG49BF5OK6NRpfc/n7vQMQiDMcz8r3NZRg6b/WQCyKf+lF8UEBoCUlBTMmyf9UWDbtm04ePCg3/7k5GQcPHgQ8fHxIdnjFrmrq0tE5DLoYlJhSJ3mckzA3xftu48FAT+2teMKrpXPhqO7KaS6SIWiAl+/fh3PPPOMpMnjjY2NePTRR2G1Wv3CXnvtNeTl5cliE8dxyMvL61/k6BQYhvyl1zPVl4H3r9A4A8uv2dPxE66VzYa9+58y1EQaFBW4ra0NCxcu9HN6sPD000/j3LlzfvuNRiM+/vhj8Ly8T3T5+fno7u7GqVOnmOHdvztFjk79S9/plfNwbLi2IbbtwZ72y7hWPhv2bvlGxKRAUYEBIDExEfn5+aJxqqqqsHbtWmbYnj17MHbsWCVMQ15eHsxms6jIfPRgRA+Zzj4d97oe/fd7/va0V+NaWR7s5vCcln1NUWy4EHC6LOvr6wU/HulwOHDPPffgzJkzfmHz5s3DoUOHlDQPALBx40Zs3bpVIJTD4OlvYtC4Z4LKu6ftIv5Zng+HWZ13Tiveg00mE0aNGoXs7Gxm+KeffoodO3b47TcYDCgtLQ34MSsY5OXlwWKx4OTJk8zw7t/LwUcnI2aI8wulrDto3/8A0NN2wSXuHwpYLQ2KCwwANTU1WL16NXQ6nddQYGdnJxYuXIibN2/6pVm7di2WLl0a0lCjlHA3Z8+e3Y/IZeANyYgeMt2Zxj1sKDBcaL1+HtfK8+GwqPtFNR2ATRC/4Q8ZLS0tyMzM9HNSbN68GWVlZX7x4+LicODAgV6XZDBDjULhQnHdIlutVtGeDI5HzND7en3PHrkBruFC87UKNP39ATgsrYJlhwkOIEwfxho+fDj99ttvvVNgv/vuO8G34RQUFHhNlxWbUy1lhaBQOqHFZy+++KJoXaLT7qGheYcp43EzZT5BlPnvRBlP2GnEgguUOGYlgRNfzhpGWoAwftouJSWFVq5cSY899hjFxsYKxispKZG0ulBIUClpxQSWIjIA4vQJZEieSIbUqaSLZb94TWV2Ahr8OOXevXtFBQ11ZaLU1YUR8B7K/vgHD0Bzn742m82SPg3rBus6LDWtL4ioN/727dsVewYPE7p4ADfUtsIXV65cYe4nCUONQugvjBUeExODzZs3i1iqedwAgEqofyrx4oQJE2RbrBYqzGYz3Xbbbaq3SZCs5AFo7tPXly5dwvHjx9U2A4Bz+U1OTo7aZgSLFh7AVbWtYGHNmjXo7OxU2wwAQGZmptomBIurPIAGta1goaamBosXL2Z6ucINvV6114mFigYeQK3aVgihvLwc9957L86cOdN7I6QGr17V5ElOCmoBwAj1bwZEyXEcPfDAA2F5YyyL6enpqrdBkDQCgAFAhwaMkcTc3Fw6evQo2e32sIhbWVmpep2DZIdLWwBAuQYMCoiTJ0+mffv2UU9Pj6ICz5kzR/W6Bslyt7g8gCINGBQUR48eTR988AF1d3czn2ODfZ52OBz00UcfqV6/EFjk0hY8gCwNGBQSR4wYQcXFxXTjxo2ARBTaPnLkCBkMBtXrFQKz4LE0iQdQoQGjQubgwYPppZde6v3oZKDDhUREhw4dEhzKjBBWgLHubIkGDJON8fHx9Oyzz1JDQwNTVCGBS0tLI11ccmnpBz2Aag0YJyujoqLoiSeeoMuXL4sOF/b09FBxcXEkfIuwP1ZDZM1ZgQYMVIQ6nY4WLFhApaWl1NLSQna7nWw2G9XW1tKOHTvozjvvVN1Gmei/sMsDPIATGjBSccbGxlJUVJTqdsjME/C59nK+OwCMAfA9gDgMIJLQBWAKgH947tTBf0Zlq4sPhMeuAciENQD+mxXA+9C9bzfUP+UMUBp3M/Tz/O3d8GQMgGMaMH6A4jzm0sr3UitptV4CBkTWMo+5NAJCeKFODAZO11rkbpc2soAHsBqASQMV+7PT5NJCkVdgGfEneU7WKE+4NAgIvjdXUliAW9CtqWFWu9o8GK2CSsTD6e9cgltkFEqjrHC1sV5MQDGdPJ0cPJzLDYXO7WJhGQAWAPgrgLsBDBKINwBx3ABQBedsjFIA9XAtAUWfPu7/bvjq4hVf6rpgz8xZhTk8fg1wDjaPBzAKwEgAtwNIBZAEp/hxcN4BGtB3dN6KcACwAbACMMPpTrwBoB3OBQdX4Zy2XAvgMoBfXHH7g6CgvhH/H3k2yZY7G4i5AAAAAElFTkSuQmCC";
```

Comment:
You have a PNG here, but an SVG file provided. If you can use an SVG here it'll save a ton of bytes over the wire.